jobs/upload-dashboards-objects/templates/dashboards-objects/dashboard/Services-DomainSearch.json.erb

```diff
@@ -11,7 +11,7 @@
         "title": "Services - Elasticsearch/OpenSearch",
         "version": 1
     },
-    "id": "domain-dashboard",
+    "id": "services-elasticsearchandopensearch",
```

Comment:
We call this offering `aws-elasticsearch` in our service catalog & on our public site; what do you think about using that instead?

Reply:
That works, This is what shows up in the url and stops duplicates